not-published/android/using-auth-sdk.adoc

```diff
@@ -337,5 +337,16 @@ You can use the `AuthService#deleteTokens` function to delete the local authenti
 
 *Note:* To perform backchannel or federated logouts, you must enable the Backchannel Logout option for the federated identity provider. More information is available in the Keycloak documentation under  http://www.keycloak.org/docs/latest/server_admin/index.html#openid-connect-v1-0-identity-providers[OIDC Identity Providers].
 
```

Comment:
@pmacko1 could you edit this please?

Reply:
yep, done

Reply:
@wtrocki Note that this doc is not actually published, ie it got moved to this repo, but is not in the published struture :(

Reply:
@finp so where we should add this then?